src/QsCompiler/CompilationManager/DataStructures.cs

```diff
@@ -23,6 +23,15 @@ namespace Microsoft.Quantum.QsCompiler.CompilationBuilder.DataStructures
     /// </summary>
     internal class CodeLine
     {
+        internal enum StringContext
+        {
+            NoOpenString,
+            OpenInterpolatedArgument,
+            OpenString,
```

Comment:
The order is important here. Suggest making a comment about this.

Reply:
I've added that to the summary for the enum.